generators/app/templates/test/index.js

```diff
@@ -18,7 +18,7 @@ describe("<%= description %>", () => {
 
     it(`should ${caseName.split("-").join(" ")}`, () => {
       const actual = babel.transformFileSync(actualFile, {
-        plugins: [plugin]
+        plugins: [plugin(babel)]
       }).code;
```

Comment:
But it will be the same file repeating in every folder right? This is the node api from babel